packages/gatsby-image/src/index.js

```diff
@@ -568,6 +597,7 @@ class Image extends React.Component {
                   alt,
                   title,
                   loading,
+                  imgStyle,
```

Comment:
If the fixed/fluid components instead move this `noscript` tag into it's own component, it could be a wrapper component to `Img`(or rather the React equivalent picture element also moved out into it's own component, a third component would take in the props to pass to each, along with the two conditional props to control state, `generateImageSources()` would only need to be called once then).

Not too important, a follow-up PR could do that.